esmtools/tests/test_temporal.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+import xarray as xr
+
+from esmtools.temporal import to_annual
+
+
+@pytest.mark.parametrize('time_type', ('datetime', 'cftime'))
+def test_to_annual(gridded_da_datetime, gridded_da_cftime, time_type):
+    """General checks that `to_annual` time conversion is working as expected."""
+    data = eval(f'gridded_da_{time_type}')()
```

Comment:
It is also possible to parametrise fixtures. See https://github.com/bradyrx/climpred/blob/7c921b2d62d15ee720c2099cedcc2254191ad86b/climpred/tests/test_bootstrap.py#L221